fbc-integration-test/catalog-source/release/managed-workspace/release_strategy.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleaseStrategy
+metadata:
+  name: fbc-strategy
+  namespace: managed-release-team-tenant
+spec:
+  pipeline: fbc-release
+  bundle: quay.io/scoheb/pipeline-fbc-release:latest
+  policy: fbc-policy
+  serviceAccount: release-service-account
+  params:
+    - name: fromIndex
+      ## staging
+      # value: "quay.io/rh-osbs/iib-pub-pending:v4.13"
+      ## prod
+      # value: "quay.io/rh-osbs/iib-pub:v4.13"
+      value: "quay.io/scoheb/fbc-index-testing:latest"
+    - name: targetIndex
+      ## staging
+      # value: quay.io/redhat-pending/redhat----redhat-operator-index:v4.13
+      ## prod
+      # value: quay.io/redhat/redhat----redhat-operator-index:v4.13
+      value: "quay.io/scoheb/fbc-target-index-testing:latest"
+      # Signing
+    - name: requester
```

Comment:
The requester and its value can be removed.

Reply:
this all irrelevant and removed, since we are running on RHTAP stage so CRs should be deployed on Stage backend